### DISCUSSION TITLE
Error running process: G:\stable-diffusion-webui-master\extensions\sd-webui-controlnet\scripts\controlnet.py

### DISCUSSION THREAD

**Discussion by Ramoyald** [Mikubill/sd-webui-controlnet#1451] [Q&A]
Updated my tool to latest version and now controlnet doesn't seem to want to play nice anymore. Each time I try to use it I'm getting this error message and I'm not sure how to fix it. I usually find a fix by myself but this one is a bit too complex for me.

Error running process: G:\stable-diffusion-webui-master\extensions\sd-webui-controlnet\scripts\controlnet.py
Traceback (most recent call last):
  File "G:\stable-diffusion-webui-master\modules\scripts.py", line 418, in process
    script.process(p, *script_args)
  File "G:\stable-diffusion-webui-master\extensions\sd-webui-controlnet\scripts\controlnet.py", line 1073, in process
    model_net = self.load_control_model(p, unet, unit.model, unit.low_vram)
  File "G:\stable-diffusion-webui-master\extensions\sd-webui-controlnet\scripts\controlnet.py", line 734, in load_control_model
    model_net = self.build_control_model(p, unet, model, lowvram)
  File "G:\stable-diffusion-webui-master\extensions\sd-webui-controlnet\scripts\controlnet.py", line 761, in build_control_model
    state_dict = load_state_dict(model_path)
  File "G:\stable-diffusion-webui-master\extensions\sd-webui-controlnet\scripts\utils.py", line 10, in load_state_dict
    if extension.lower() == ".safetensors":
AttributeError: 'tuple' object has no attribute 'lower'

Any help on how to fix this would be great and thanks for making incredible tool for SD :) 
ControlNet v1.1.200

**Comment by Jackyutopia** [comment]
I have the same problom，how can i deal with it?

**Comment by AL3X830** [comment]
same problem

**Comment by Guillaume-Fgt** [comment]
@Jackyutopia and @AL3X830 , do you have exactly the same error message? The line number is not the same between the repo code and Ramoyald error message for the `build_control_model()` function in `controlnet.py`: 761 VS 311. I find it strange.

Also, can you give a list of the filenames of models you have in your `\stable-diffusion-webui\models\ControlNet` folder and the selected ControlNet model when the error happens.

**Reply by Ramoyald** [reply]
this folder is actually empty, my controlnet models are located in `\stable-diffusion-webui\extensions\sd-webui-controlnet\models` which contains 

ip-adapter_sdxl.bin
kohya_controllllite_xl_canny.safetensors
kohya_controllllite_xl_dept.safetensors
sai_xl_canny_256lora.safetensors
t2i-adapter_diffusers_xl_openpose.safetensors

**Comment by Ramoyald** [comment]
Although I still have no idea what caused the issue, I decided to simply reinstall everything in a different folder. Now it works normally.
